recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 974480e2cee98071470ce8ca13eab600a32f213310b9fdc14a0b15ab6d50cb07
```

Comment:
This probably need changing too!

Reply:
Oops, yeah. I'm a newbie at this conda feedstock stuff!

Reply:
Hash updated.